cmd/sovereignnode/sovereignNodeRunner.go

```diff
@@ -0,0 +1,1715 @@
+package main
```

Comment:
Please add a TODO to have unit tests here. I have done a couple of unit tests that are merged in the RC/v1.5.0 branch. Also, I think we can have a base component for the nodeRunner in order to avoid unnecessary duplicated code (example: functions `printEnableEpochs`, `doProfileOnShuffleOut`, `addSyncersToAccountsDB` and almost all static functions)

Reply:
Right, good point.
Added `TODO`